src/DropDownMenu/DropDownMenu.js

```diff
@@ -148,9 +134,9 @@ const DropDownMenu = React.createClass({
 
   getStyles() {
     const {disabled} = this.props;
-    const spacing = this.state.muiTheme.rawTheme.spacing;
-    const palette = this.state.muiTheme.rawTheme.palette;
-    const accentColor = this.state.muiTheme.dropDownMenu.accentColor;
+    const spacing = this.context.muiTheme.rawTheme.spacing;
```

Comment:
What the heck?! I thought we got rid of all of our internal references to `rawTheme` already? I guess not :fearful: